From fd1dd19322bd8fe44cca880d9a9ab8c4b0cf06fc Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Mon, 14 Nov 2016 15:21:07 -0600
Subject: [PATCH 1/5] Revert "git.js doesn't need amd-loader and it's causing a
 global variable leak which makes mocha tests fail"

This reverts commit 5d7d4e3b3dab1d781a2b171cf2e75e75b02d483f.

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 node_modules/c9/git.js | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/node_modules/c9/git.js b/node_modules/c9/git.js
index 6228914a..dea02285 100644
--- a/node_modules/c9/git.js
+++ b/node_modules/c9/git.js
@@ -1,5 +1,7 @@
 "use strict";
 
+require("amd-loader");
+
 var Fs = require("fs");
 var Path = require("path");
 var exec = require("child_process").exec;
-- 
2.11.0

